enums.h

```diff
@@ -2878,3 +2878,10 @@ typedef enum action_id {
 	ACTION_SPELLPLR    = 25,
 	ACTION_SPELLWALL   = 26,
 } action_id;
+
+typedef enum dlrg_flag {
+	DLRG_HDOOR = 0x01,
+	DLRG_VDOOR = 0x02,
+	DLRG_CHAMBER = 0x04,

```

Comment:
Should have been 0x40 ;)

Reply:
Shoot, do you need another PR or something?

Reply:
No worries, already fixed it :)
https://github.com/diasurgical/devilution/commit/a52d36a90fcdfb4029d29f54240cf02aa9939856